package.json

```diff
@@ -4,11 +4,11 @@
   "description": "email notifier to monitor the notificaion kafka topic and send emails.",
   "main": "app.js",
   "dependencies": {
-    "@hapi/hapi": "18.3.1",
+    "@hapi/hapi": "^18.4.0",
     "@mojaloop/central-services-error-handling": "7.3.0",
+    "@mojaloop/central-services-health": "^8.3.0",
```

Comment:
please remove caret i.e. ^

Reply:
Hey @jgeewax I'm a little confused as to your comment above, what tooling would you want to fix? and what invalid results would audit:resolve spit out?

Reply:
The output that you're telling me to change (with the `^` characters) is the output of `npm run resolve:audit`... If these values are forbidden by our policies, then the tool should probably not produce them, right? @rmothilal

Reply:
no that is the output of `npm run dep:check`

Reply:
and that isnt a library that we own

Reply:
Right... what I'm saying is... if we have rules that say "no carets please", can't we just enforce that with a short script?

E.g., After the audit fixes are applied to package.json, we can automagically run a quick `sed 's/: "^/: "/'` ?

Reply:
so you suggest something like `"clean:package": "sed -i.bak 's/: \"^/: \"/' package.json"` in all package.json files?

Reply:
I've added this to the npm resolve:audit line in package.json. But basically, yes. If we don't allow carets, we should make code do that for us rather than running a script and having the output of that script be considered invalid by the team.

Reply:
@jgeewax here's a discussion on the versioning convention we use - https://github.com/mojaloop/documentation/blob/master/contributors-guide/standards/versioning.md